src/components/pages/advanced.tsx

```diff
@@ -56,24 +88,17 @@ function Advanced(props:AdvancedProps): JSX.Element {
               <div>False</div>
               <div>True</div>
             </div>
-
           </div>
         </div>
-
         <h1>Select the matching robot:</h1>
         <div className="robots">
-          <button className="robot-button">
-            <img className="robot-image" src={robot1} alt="red robot, circle head, with mouth, 1 antenna, 2 arms, 1 button"></img>
-          </button>
-          <button className="robot-button">
-            <img className="robot-image" src={robot2} alt="red robot, square head, with mouth, 1 antenna, 0 arms, 1 button"></img>
-          </button>
-          <button className="robot-button" onClick={props.onCorrect}>
-            <img className="robot-image" src={robot3} alt="blue robot, triangle head, with mouth, 2 antenna, 2 arms, 3 buttons"></img>
-          </button>
-          <button className="robot-button">
-            <img className="robot-image" src={robot4} alt="green robot, circle head, with mouth, 2 antenna, 2 arms, 1 button"></img>
-          </button>
+          {robotImages.map((robot, index) => (
+            <Reward key='robot${index}' ref={refList[index]} type="confetti" config={{angle: 70, lifetime: 125}}>
+              <button className="robot-button" onClick={() => robots(index)}>
+                <img className="robot-image" src={robot} alt={getAltText(index)}></img>
+              </button>
+            </Reward>
```

Comment:
Wrapping the reward component in a div is needed since reward is made up of multiple elements which causes some alignment issues when the robots take up two columns.
```suggestion
          <div>
            <Reward key='robot${index}' ref={refList[index]} type="confetti" config={{angle: 70, lifetime: 125}}>
              <button className="robot-button" onClick={() => robots(index)}>
                <img className="robot-image" src={robot} alt={getAltText(index)}></img>
              </button>
            </Reward>
         </div>
```